Description: Upstream changes introduced in version 0.0.0-0~rzr2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 p-uae (0.0.0-0~rzr2) oneiric; urgency=low
 .
   * wip
 .
 The person named in the Author field signed this changelog entry.
Author: Philippe Coval <rzr@gna.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- p-uae-0.0.0.orig/p-uae.spec
+++ p-uae-0.0.0/p-uae.spec
@@ -13,7 +13,7 @@ Name: p-uae
 Version: 0.0.0
 Release: 0
 URL: http://sourceforge.net/projects/uaedev/
-Source0: %{name}-%{version}.orig.tar.gz
+Source0: %{name}_%{version}.orig.tar.gz
 Source1: %{name}.desktop
 License: GPL
 Group: Emulators
